third_party/amd/lib/TritonAMDGPUTransforms/StreamPipeline.cpp

```diff
@@ -72,6 +72,12 @@ namespace {
 // 1. The user provides a `num_stages` that specifies how many stages the
 //    pipeline will have. The number of stages must be larger than the distance
 //    from the first independent load to the compute in order to pipeline.
+//    1.a. User may also specify `local_store_stage=<s>` to set the stage
+//         for ttg.local_store ops create by the pipeliner.
+//    1.b. User may also specify `local_load_stage=<s>` to set the stage
```

Comment:
These two need to be change to `TRITON_HIP_GLOBAL_PREFETCH` and `TRITON_HIP_LOCAL_PREFETCH` to match the definition in compiler.py

Reply:
Yes, I have re-aligned to this. It is the simplest to convey.